app/src/features/apiClient/screens/apiClient/components/modals/MoveToCollectionModal/MoveToCollectionModal.tsx

```diff
@@ -56,19 +74,27 @@ export const MoveToCollectionModal: React.FC<Props> = ({ isOpen, onClose, record
     }
   }, [user?.details?.profile?.uid, team?.id, onSaveRecord, selectedCollection?.label]);
 
-  const moveRecordToCollection = useCallback(
+  const moveRecordsToCollection = useCallback(
     async (collectionId: string, isNewCollection: boolean) => {
-      const updatedRequest = { ...recordToMove, collectionId };
-      const result = await upsertApiRecord(user?.details?.profile?.uid, updatedRequest, team?.id);
-      if (result.success) {
+      const updatedRequests = recordsToMove.map((record) =>
+        isApiCollection(record)
+          ? { ...record, collectionId, data: omit(record.data, "children") }
+          : { ...record, collectionId }
+      );
+
```

Comment:
_same as the Object.assign comment for duplication_
this does not update the `updateTs` and _possibly_ other relevant (if/any) fields of record metadata